vllm/worker/model_runner.py

```diff
@@ -515,6 +531,12 @@ def _prepare_model_input(
                 seq_lens.append(1)
                 block_tables.append([])
                 lora_index_mapping.append(0)
+
+                last_paged_kv_indptr = paged_kv_indptr[-1]
```

Comment:
paged_kv_indptr is None if flash infer is not used? I think this should be inside if statement?

Reply:
Good catch, thanks!